dictionary.go

```diff
@@ -304,244 +304,530 @@ var colors = []string{
 
 // Word corpus of animals.
 var animals = []string{
+	"aardvark",
 	"albatross",
 	"alligator",
 	"alpaca",
 	"anchovy",
-	"ant",
```

Comment:
I see a couple of existing entries deleted, any particular reason behind this?

Reply:
not really. I only grabbed a list of animals I found online and replaced the existing list with it. I will merge the old and new lists